zilencer/management/commands/populate_db.py

```diff
@@ -269,6 +282,13 @@ def add_arguments(self, parser: CommandParser) -> None:
             help="The percent of messages to be personals.",
         )
 
+        parser.add_argument(
+            "--percent-attachments",
+            type=float,
+            default=15,
```

Comment:
15% seems rather high for this?

Also, we might want to separate "messages" from "images"?

Reply:
I will change it to 5%. Can you clarify the comment "separate messages from images"? The attachment is a temp file with random text, and they are embedded in some % of generated messages. Thanks.

Reply:
@timabbott Sorry to ping you. I would like to address your comment about "separate messages from images." When you get the chance, can you help me clarify? Thanks!

Reply:
Sorry, was a typo -- I meant "attachments" might be a separate bucket from "images", since not every attachment is an image.

Reply:
The existing code inserts random images into messages. This PR will attach a text file to a percentage of messages, so I think images and attachments are separate.